lib/phobos/producer.rb

```diff
@@ -21,18 +21,20 @@ def initialize(host_obj)
         @host_obj = host_obj
       end
 
-      def publish(*args, **kwargs)
-        Phobos.deprecate(deprecate_positional_args_message('publish')) if kwargs.empty?
-
-        args = normalize_arguments(*args, **kwargs)
-        class_producer.publish(**args)
+      def publish(topic:, payload:, key: nil, partition_key: nil, headers: nil)
```

Comment:
We can also remove the `MissingRequiredArgumentsError` exception class 🔥

Reply:
Done!